product_pack/models/product_product.py

```diff
@@ -0,0 +1,102 @@
+# Copyright 2019 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models, api
+import math
+
+
+class ProductProduct(models.Model):
+    _inherit = 'product.product'
+
+    pack_line_ids = fields.One2many(
+        'product.pack.line',
+        'parent_product_id',
+        'Pack Products',
+        help='List of products that are part of this pack.'
```

Comment:
```suggestion
        help='Products that are part of this pack.'
```

This might be a list, or it could be a kanban... Well, they're products anyway.